src/core/recv_buffer.h

```diff
@@ -122,12 +140,24 @@ QuicRecvBufferHasUnreadData(
 // Changes the buffer's virtual buffer length.
 //
 _IRQL_requires_max_(DISPATCH_LEVEL)
-void
-QuicRecvBufferIncreaseVirtualBufferLength(
+BOOLEAN
```

Comment:
Do we really need to return the BOOLEAN? If no one uses it, let's just drop it.

Reply:
The return value will be used by the caller - mostly for diagnostic (avoiding confusing logs about increasing a buffer length when it actually stays the same).